apps/src/templates/sectionProgressV2/InviteToV2ProgressModal.jsx

```diff
@@ -36,6 +36,14 @@ function InviteToV2ProgressModal({
   const [invitationOpen, setInvitationOpen] = React.useState(false);
 
   React.useEffect(() => {
+    if (
+      dateProgressTableInvitationDelayed === undefined ||
+      hasSeenProgressTableInvite === undefined
+    ) {
+      // Do not proceed if data has not been fully loaded.
```

Comment:
Intentionally left this comment in here.

Reply:
Note that once this data is fully loaded (for even a brand new user), the values are set to `null` and `false`

<img width="370" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/24883357/f059961a-59f7-45de-ab89-adf1dc65bc1e">